src/Tizen.NUI/src/internal/Transition/TransitionSetSignal.cs

```diff
@@ -0,0 +1,86 @@
+ */
+
+namespace Tizen.NUI
+{
+    internal class TransitionSetSignal : Disposable
+    {
+        internal TransitionSetSignal(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
+        {
+        }
+
+        protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef swigCPtr)
+        {
+            Interop.TransitionSet.DeleteTransitionSetSignal(swigCPtr);
+        }
+
+        public bool Empty()
+        {
+            bool ret = Interop.TransitionSet.TransitionSetSignalEmpty(SwigCPtr);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return ret;
+        }
+
+        public uint GetConnectionCount()
+        {
+            uint ret = Interop.TransitionSet.TransitionSetSignalGetConnectionCount(SwigCPtr);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return ret;
+        }
+
+        public void Connect(System.Delegate func)
+        {
+            System.IntPtr ip = System.Runtime.InteropServices.Marshal.GetFunctionPointerForDelegate<System.Delegate>(func);
+            {
+                Interop.TransitionSet.TransitionSetSignalConnect(SwigCPtr, new System.Runtime.InteropServices.HandleRef(this, ip));
+                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            }
+        }
+
+        public void Connect(System.IntPtr callback)
+        {
+            Interop.TransitionSet.TransitionSetSignalConnect(SwigCPtr, new System.Runtime.InteropServices.HandleRef(this, callback));
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        public void Disconnect(System.Delegate func)
+        {
+            System.IntPtr ip = System.Runtime.InteropServices.Marshal.GetFunctionPointerForDelegate<System.Delegate>(func);
+            {
+                Interop.TransitionSet.TransitionSetSignalDisconnect(SwigCPtr, new System.Runtime.InteropServices.HandleRef(this, ip));
+                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            }
+        }
+
+        public void Disconnect(System.IntPtr callback)
+        {
+            Interop.TransitionSet.TransitionSetSignalDisconnect(SwigCPtr, new System.Runtime.InteropServices.HandleRef(this, callback));
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        public void Emit(TransitionSet arg)
+        {
+            Interop.TransitionSet.TransitionSetSignalEmit(SwigCPtr, TransitionSet.getCPtr(arg));
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        public TransitionSetSignal() : this(Interop.TransitionSet.NewTransitionSetSignal(), true)
```

Comment:
Not needed? It links to a floating signal, not attached to any object...

Reply:
Thank you I removed it.